src/components/input/input.js

```diff
@@ -80,11 +80,20 @@ function mdInputContainerDirective($mdTheming, $parse) {
 
   var INPUT_TAGS = ['INPUT', 'TEXTAREA', 'SELECT', 'MD-SELECT'];
 
+  var HIDDEN_INPUT_SELECTORS = ':not([hidden=""])'
```

Comment:
I'm not a fan of all those selectors.. can't we use siblings or something else instead of this list?